packages/mask/src/plugins/NextID/locales/zh-CN.json

```diff
@@ -1 +1,35 @@
-{}
+{
+    "wallet": "钱包",
```

Comment:
Please make sure your i18n files are contributed on Crowdin, otherwise it will be auto removed in the regular sync with Crowdin.